arcgis-ios-sdk-samples/Search/Reverse geocode/README.md

```diff
@@ -25,6 +25,10 @@ Tap the map to see the nearest address displayed in a callout.
 * AGSLocatorTask
 * AGSReverseGeocodeParameters
 
+## About the data
+
+This sample uses the [World Geocoding Service](https://www.arcgis.com/home/item.html?id=305f2e55e67f4389bef269669fc2e284).
```

Comment:
One question for the 3 Geocoding service URL changes in READMEs above: do we also need to change the URL in the sample?

Reply:
I think this is related to /common-samples/issues/2246

Reply:
Yes, I think you're correct. The sample source code will be changed in a separate PR.